### ISSUE TITLE
bugs, datasets and mismatch of experimental results

### ISSUE THREAD

**Issue by yuanhangtangle** [CuteyThyme/RISK#3]
1. I found some **bugs** such as
```python
test_acc = rsr_validate(test_loader, model, criterion, epoch, True, args, datasetname=args.dataset)
```
where the `datasetname` argument should be 'HANS' when evaluating on HANS dataset.

2. I downloaded the datasets following the instruction in `README`, but the **prepocessing functions in `src` fail to process those files due to the mismatch of file format**

3. The original paper does NOT provide a standard split and I can't find a publicly available split of the QQP dataset. **Could you please provide your train-dev-test splits?**

4. I fixed the bugs and tried to reproduce the result of MNLI, **but my test accuracy on HANS is only around 62.** I don't know what goes wrong ...

And finally, I will appreciate it if you can update your code and readme file as well as providing your split of the QQP dataset.

**Comment by CuteyThyme** [comment]
Yes, I have found that when I uploaded the codes, I made some mistakes here( thanks for your check~). Recently, I reproduce the results in the server under different environments(pytorch/GPU versions), I found these factors can have impacts on the results. For the results in my paper, I suggest torch version with 1.7.1 and NVIDIA GeForce RTX 2080 Ti.
